sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/component.py

```diff
@@ -322,7 +325,8 @@ def _try_build_local_code(self) -> Iterable[Code]:
         # origin code value of internal component will never be None. check _get_origin_code_value for details
         with self._generate_additional_includes_obj().merge_local_code_and_additional_includes() as tmp_code_dir:
             # use absolute path in case temp folder & work dir are in different drive
-            tmp_code_dir = tmp_code_dir.absolute()
+            tmp_code_dir_path: Path = tmp_code_dir
+            tmp_code_dir_path = tmp_code_dir_path.absolute()
```

Comment:
```suggestion
            tmp_code_dir_path = tmp_code_dir.absolute()
```
should be enough?
That said, is this new variable used for anything?